src/lib/components/explorer/export/ExportStepper.svelte

```diff
@@ -119,11 +119,10 @@
 
   async function download(): Promise<void> {
     try {
-      const res = await api.post(`picsure/query/${datasetId}/result`, {});
-      const responseDataUrl = URL.createObjectURL(new Blob([res], { type: 'octet/stream' }));
+      const signedUirl = await getSignedUrl();
```

Comment:
isn't `download()` used for csv as well? We don't access those via a signed URL, right? Also, isnt PFB supposed to work without a signed URL by also downloading it the same way we did with csv's?

Reply:
Yes, but downloading the same way we do CSVs messes up the PFB encoding